### PR TITLE
Feat: Implements and Styles footer

### PR DESCRIPTION
**Pull request by saranicoly**
## What's the purpose of this pull request?
<em>The footer is a mix of many implemented components. It can be seen here:</em>

|Desktop|
|-|
Desktop

|Mobile|
|-|
Mobile

## How it works? 
<em>It uses the following components: Link, Icon, Accordion, AccordionItem, AccordionButton, AccordionPanel, Incentive and Payment Methods.</em>

## How to test it?
Visit the preview [link](https://preview-177--base.preview.vtex.app).

## References

[Jira issue FSSS-79 Create Footer Component](https://vtex-dev.atlassian.net/jira/software/projects/FSSS/boards/632?selectedIssue=FSSS-59)
[Jira issue FSSS-78 Style Footer Component](https://vtex-dev.atlassian.net/jira/software/projects/FSSS/boards/632?selectedIssue=FSSS-60)

### CODE REVIEW

File: src/components/common/Footer/LinksList.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  Icon,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+} from '@faststore/ui'
+
+import { PlusButtonIcon } from './Icons'
+
+function LinksList() {
```
Comment (hellofanny) [inline src/components/common/Footer/LinksList.tsx:13]: Maybe we can just call it `FooterLinks`

Comment (saranicoly) [reply src/components/common/Footer/LinksList.tsx:13]: Great suggestion! I'm going to change it.

File: src/components/common/Footer/footer.scss [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  [data-store-list] {
+    display: flex;
+    justify-content: center;
+    margin-top: var(--space-4);
+    li:not(:last-child) { margin-right: var(--space-6); }
+  }
+}
+
+.footer__note,
+.footer__section {
```
Comment (renatamottam) [inline src/components/common/Footer/footer.scss:33]: We can try to use css grid here 🙃

**Review by renatamottam — COMMENTED** [review]
Good job, Fan and Sara! I left a couple suggestions :)

**Review by filipewl — APPROVED** [review]
Blessed be.

**Review by lucasfp13 — APPROVED** [review]

**Comment by hellofanny** [comment]
OMGGG I CAN'T BELIEVEEE 🥳 🥳 🥳 🥳  THANKS TEAM!!!!

**Review by hellofanny — APPROVED** [review]

**Comment by heitorado** [comment]
Ladies and gentlemen, the merge!

**Pull request merged**